SDSetupBlazor/Pages/Consoles.cshtml

```diff
@@ -249,7 +249,7 @@
 
 
 			HttpClient client = new HttpClient();
-			client.Timeout = TimeSpan.FromSeconds(120);
+			client.Timeout = TimeSpan.FromSeconds(200);
```

Comment:
And here

Reply:
I thought I rolled back/extracted this change from this PR.  I'll fix that.